src/utils/utils.cpp

```diff
@@ -342,3 +342,10 @@ void mp::utils::check_and_create_config_file(const QString& config_file_path)
         config_file.open(QIODevice::WriteOnly);
     }
 }
+
+// Given an absolute path, removes trailing slash, multiple slashes and
+// redundant "." and "..".
+std::string mp::utils::normalize_absolute_path(const std::string& path)
+{
+    return QDir::cleanPath(QString::fromStdString(path)).toStdString();
```

Comment:
Nice, `QDir::cleanPath` indeed seems the right choice for this. Symlinks are left alone, but that is probably ok, and if we decide we should resolve them, that can always done in a separate PR.

Reply:
Sure. Symlinks can be resolved exactly as we executed 'pwd' when mounting/unmounting. We'll see that later.

Reply:
Right, something along those lines, like `pwd -P` or `readlink -f`. Something to think about.